### DISCUSSION TITLE
ics.py 0.8.0.dev0  installs version 0.7.2. Why?

### DISCUSSION THREAD

**Discussion by spencer802** [ics-py/ics-py#409] [Q&A]
I have downloaded the code as .zip and installed it using pip install ics, but the status at the end says "successfully installed ics-0.7.2".

I have been having trouble using some suggested code that may require 0.8.0, but I cannot install it, evidently.

**Comment by C4ptainCrunch** [comment]
This seems very weird indeed. Could you paste all the commands you run + their full output so we can try to reproduce ?
